src/domains/checkout-app/99_main.tf

```diff
@@ -0,0 +1,39 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "<= 3.40.0"
```

Comment:
Maybe here is better to use a fixed version of the provider instead of a version range?

Reply:
Seems to be devops' recommendation, see e.g. https://github.com/pagopa/io-infra/pull/411